python/utilities/log_list/print_log_list.py

```diff
@@ -131,6 +134,75 @@ def generate_code_for_chromium(json_log_list, output_file):
     f.write("\nconst size_t kNumKnownCTLogs = %d;\n" % len(logs))
     write_cpp_footer(f, include_guard)
 
+def write_android_header(f):
```

Comment:
While you're at it, please look into breaking the C++ code generation to a module of its own.